src/lib/pages/account-details/components/tables/txs/TxsTable.tsx

```diff
@@ -88,7 +78,7 @@ export const TxsTable = ({
     refetch: refetchTxsCount,
     isLoading: txsCountLoading,
     failureReason,
-  } = useTxsCountByAddress(walletAddress, accountId, "", filters, isSigner);
+  } = useTxsCountByAddress(undefined, accountId, "", filters, isSigner);
```

Comment:
Would it be possible to pass object arguments instead of several individual arguments?